### PR TITLE
[Fix] Updated new polygon icon svg

### LINKED ISSUE #1068: [BUG] Polygon. Icon bug
**Describe the bug**

Current Polygon icon SVG renders incorrectly.

**To Reproduce**
https://oui.opensearch.org/1.3/#/display/icons (Look for Polygon

**Expected behavior**

Items are filled when they should be empty.

**Screenshots**
Current Incorrect:
Screenshot 2023-10-09 at 9 14 58 AM

Should render correctly like below:
Screenshot 2023-10-09 at 9 19 34 AM

**Replace with SVG below**
[polygon.zip](https://github.com/opensearch-project/oui/files/12848358/polygon.zip)

### PR DESCRIPTION
**Pull request by Willie-The-Lord**
### Description

Updated new polygon icon svg 

### Issues Resolved

- Updated svg
- Updated related snapshots
- Fixes #1068 

#### Before
before_light
before_dark

#### After
after_light
after_dark

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn run test-unit icon -u`
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/components/icon/assets/polygon.svg [hunk 1]
```diff
@@ -4,5 +4,6 @@
   -->
 
 <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <path d="M1 5V1H5V2.29999H11V1H15V5H12.0658L11.2581 6H12V10H11.2403L12.0403 11H15V15H11V13.5H5V15H1V11H2.5V5H1ZM2 2H4V4H2V2ZM11 12.5V11.3004L9.95972 10H8V6H9.97266L11 4.72806V3.29999H5V5H3.5V11H5V12.5H11ZM12 2H14V4H12V2ZM4 12H2V14H4V12ZM9 7H11V9H9V7ZM12 12H14V14H12V12Z"/>
-</svg>
+  <path
+    d="M15 11v4h-4v-1.5H5V15H1v-4h1.5V5H1V1h4v1.3h6V1h4v4h-2.9l-.8 1h.7v4h-.8l.8 1h3zm-1 3v-2h-2v2h2zm0-10V2h-2v2h2zm-3 8.5v-1.2L10 10H8V6h2l1-1.3V3.3H5V5H3.5v6H5v1.5h6zM11 9V7H9v2h2zm-7 5v-2H2v2h2zM4 4V2H2v2h2z" />
+</svg>
```
Comment (BSFishy) [inline src/components/icon/assets/polygon.svg:9]: Can we get a blank line at the end of the file?

Comment (Willie-The-Lord) [reply src/components/icon/assets/polygon.svg]: No problem.

**Comment by Willie-The-Lord** [comment]
@BSFishy I've added the blank line. Thanks!

**Review by BSFishy — APPROVED** [review]

**Comment by Willie-The-Lord** [comment]
@BSFishy Just a little curious about the reason of adding a blank space. Is it for maintaining consistency with other SVG files?

**Review by joshuarrrr — APPROVED** [review]

**Comment by joshuarrrr** [comment]
@Willie-The-Lord The blank line at the end of every file is a more general convention - see https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file . For the purpose of this project, it's mostly about ensuring clean diffs via git.

**Comment by Willie-The-Lord** [comment]
@joshuarrrr Thank you for sharing!

**Pull request merged**